src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -174,6 +190,41 @@ function useSidebarMetadataFetcher(
         [api, file, onApiError],
     );
 
+    const extractSuggestions = React.useCallback(
+        async (templateKey: string, fields: MetadataTemplateField[]) => {
+            const aiAPI = api.getIntelligenceAPI();
+
+            await wait(1000);
```

Comment:
what's the reasoning for this? are we adding an artificial loading time?

Reply:
sorry, I was using it to test loading state, and forgot to delete